### ISSUE TITLE
[Bug]  Xamarin Forms Accessibility for the Layout<View> child ContentView label text not working.

### ISSUE THREAD

**Issue by jeyakasipandi** [xamarin/Xamarin.Forms#13796]
### Description

We have created a custom Layout<View> to add the ContentView Label when select the view label text not accessible.

[XAML]

```
    <local:CustomView AutomationId="CustomLayout" VerticalOptions="Center"  HeightRequest="300" WidthRequest="300" BackgroundColor="Aqua" >
        <ContentView BackgroundColor="Yellow" HeightRequest="300" WidthRequest="300">
            <Label Text="LabelText" FontSize="25" HorizontalOptions="CenterAndExpand" VerticalTextAlignment="Center"/>
        </ContentView>
    </local:CustomView>
```

[C#]

```
    public class CustomView : StackLayout
    {
        public CustomView()
        {
            this.SetValue(AutomationProperties.IsInAccessibleTreeProperty, true);

            AutomationProperties.SetName(this, this.AutomationId + "LayoutItem 1 of 1");
        }
}
```

If enable IsInAccessibleTreeProperty for both the CustomView and ContentView. The voice over will be only Label text.
 
### Expected Behavior

While select the view the voice over should be LabelText + LayoutItem 1 of 1.

### Actual Behavior

The child view text not accessible.

### Basic Information

XF version 4.8

### Environment

Visual Studio 2019

### Screenshots

![image](https://user-images.githubusercontent.com/62105021/108352170-8155f400-720c-11eb-9d51-d347652dc979.png)

### Sample LInk

[Accessibilitydemo.zip](https://github.com/xamarin/Xamarin.Forms/files/6002593/Accessibilitydemo.zip)

**Comment by rachelkang** [comment]
Hi, @jeyakasipandi - thanks for submitting this issue. I'm confirming I'm able to reproduce this behavior on both Android and iOS, even on the latest version of XF 5.0.0.2012.

As you described - For some reason, the default is that the child subview is accessible while the parent view is not - yet, setting `IsInAccessibilityTree="True"` on the parent view is making the parent view accessible while making the child subviews inaccessible. It seems that setting `IsInAccessibilityTree="True"` directly on the child view makes it accessible again on Android, but not iOS. Further exploration is needed to understand why this is the case.

This issue is also not limited just to custom views. The same issue was reproduced without the custom view:

```xaml
<StackLayout AutomationProperties.IsInAccessibleTree="True" AutomationProperties.Name="LayoutItem 1 of 1">
     <ContentView BackgroundColor="Yellow" HeightRequest="300" WidthRequest="300">
             <Label Text="LabelText" FontSize="25" HorizontalOptions="CenterAndExpand" VerticalTextAlignment="Center" /> 
     </ContentView>
</StackLayout>
```

**Comment by rachelkang** [comment]
Hi, @jeyakasipandi - in order to achieve the before you want, you may be able to leverage the all-new [SemanticOrderView ](https://github.com/xamarin/XamarinCommunityToolkit/pull/1240)from Xamarin Community Toolkit or even use different layouts (i.e. Grid) to layer the controls.

If you haven't already, I also recommend updating to the latest version Xamarin.Forms or at least to Xamarin.Forms 5.0.0.2083, which has many accessibility improvements. These improvements include an experimental accessibility flag that addresses a lot of improvements in Android accessibility.

To leverage this flag on the latest versions of Xamarin Forms, the following code can be inserted in the App.xaml.cs file, following the InitializeComponent call and before the MainPage is set: `Xamarin.Forms.Device.SetFlags(new List<string> { "Accessibility_Experimental"});`

**Comment by jfversluis** [comment]
Assuming this is resoled with the information that was provided by us
